code/modules/mob/living/living_defense.dm

```diff
@@ -121,7 +121,7 @@
 	if(thrown_item.thrownby == WEAKREF(src)) //No throwing stuff at yourself to trigger hit reactions
 		return ..()
 
-	if(mob_run_block(AM, thrown_item.throwforce, "\the [thrown_item.name]", ATTACK_TYPE_PROJECTILE, 0, throwingdatum.thrower, throwingdatum.thrower.zone_selected, list()))
+	if(mob_run_block(AM, thrown_item.throwforce, "\the [thrown_item.name]", ATTACK_TYPE_THROWN, 0, throwingdatum.thrower, throwingdatum.thrower.zone_selected, list()))
```

Comment:
`ATTACK_TYPE_PROJECTILE` and `ATTACK_TYPE_THROWN` are both bitflags, so you could probably do
```suggestion
	if(mob_run_block(AM, thrown_item.throwforce, "\the [thrown_item.name]", (ATTACK_TYPE_PROJECTILE | ATTACK_TYPE_THROWN), 0, throwingdatum.thrower, throwingdatum.thrower.zone_selected, list()))
```

Reply:
I am aware, but it simply won't do, stuff that go down from mob_run_block here will only accept thrown, and this will only ever be thrown.